lib/src/generator/template_data.dart

```diff
@@ -99,6 +99,10 @@ abstract class TemplateData<T extends Documentable> extends TemplateDataBase {
   @override
   T get self;
 
+  String? get aboveSidebarPath;
```

Comment:
Each template now has an optional `aboveSidebarPath` and a `belowSidebarPath` which is included in the main HTML, and the JS uses those paths to request each sidebar client-side.